### ISSUE TITLE
Run child reducer on redux-undo actions as well

### ISSUE THREAD

**Issue by arekkas** [omnidan/redux-undo#129]
At the moment, redux-undo skips the child reducer when an action is triggered that is known to redux-undo, for example [here](https://github.com/omnidan/redux-undo/blob/master/src/reducer.js#L157-L161):

```jsx
      case config.undoType:
        res = undo(history)
        debug.log('perform undo')
        debug.end(res)
        return res
```

Unfortunately, I have a use case where I need to execute the reducer there as well. I am aware that this is a very specific use case which is why I think that this should be configurable:

```jsx
      case config.undoType:
        res = undo(history)
        debug.log('perform undo')
        debug.end(res)
        return config.neverSkipReducer
          ? { ...res, present: reducer(res.present, action) }
          : res
```

**Comment by MicleMing** [comment]
I am confusing about two questions
  -. why can i not config `neverSkipReducer ` 's value  to `true`, i want to deal with `undo` action in my code, so i don't want to skip the raw reducer when action is `undo`

but in `reducer.js` (redux-undo), `neverSkipReducer` is `false` and i has no chance to change it

```
  var config = {
    ...
    neverSkipReducer: false
  };
```

  - reducer's arguments
```
var skipReducer = function skipReducer(res) {
      return config.neverSkipReducer ? _extends({}, res, { present: reducer(res, action) }) : res;
 };
```
`reducer`  should  get  `res.present`  as param,  but, why it get `res` as param ?

**Comment by arekkas** [comment]
Hey @MicleMing this is primarily because I messed up. It was late when I created the PR. I already patched this in #133 but @omnidan is busy this week so we won't see it merged before next week. Sorry for that!

**Comment by omnidan** [comment]
@MicleMing @arekkas sorry for the late response - I've merged #133 now and this will be fixed in the next release (I'll try to get all PRs that are almost done into this, so you can expect this in a few days maximum)

Let me know if it works fine in `master`

**Comment by omnidan** [comment]
released `beta9-2` with this change
